keras/layers/reshaping/permute.py

```diff
@@ -62,9 +62,11 @@ def __init__(self, dims, **kwargs):
         self.dims = tuple(dims)
         if sorted(dims) != list(range(1, len(dims) + 1)):
             raise ValueError(
-                "Invalid permutation argument `dims` for Permute Layer. "
-                "The set of indices in `dims` must be consecutive and start "
-                f"from 1. Received dims={dims}"
+                "Invalid permutation `dims` for Permute Layer: %s. "
+                "`dims` must be a permutation of the input's axis ids, "
```

Comment:
I would not refer to these as _axis ids_. These are simply the indices of the axes (as opposed to some identifiers).

How about:

> `dims` must be a permutation of the input's axis indices,